test/integration/options.js

```diff
@@ -466,6 +466,61 @@ module.exports = function (createFn, setup, dismantle) {
     })
   })
 
+  describe('generateMethods', () => {
+    let app = createFn()
+    let server
+
+    before(done => {
+      setup(err => {
+        if (err) {
+          return done(err)
+        }
+
+        erm.serve(app, db.models.Customer, {
+          generateMethods: ['get'],
+          restify: app.isRestify
+        })
+
+        server = app.listen(testPort, done)
+      })
+    })
+
+    after(done => {
+      dismantle(app, server, done)
+    })
+
+    it('GET /Customer 200', done => {
```

Comment:
Codacy has a fix for the issue: [Replace `done` with `(done)`](https://app.codacy.com/gh/florianholzapfel/express-restify-mongoose/pullRequest?prid=8423228)
```suggestion
    it('GET /Customer 200', (done) => {
```